### PR TITLE
Formatting

### PR DESCRIPTION
**Pull request by Tech-TTGames**
Various formatting changes to fit formatting conventions.

### CODE REVIEW

File: SUPPORT.md [hunk 1]
```diff
@@ -1,3 +1,5 @@
+# What to do?
```
Comment (kk5dire) [inline SUPPORT.md:1]: Maybe make the text smaller?

Perhaps replace with something more along the lines of "What should I do?"

**Review by kk5dire — APPROVED** [review]
LGTM

See review comment history for some suggestions before merging

**Comment by kk5dire** [comment]
Resolve conversation when ready to merge, added suggestions in discussion

File: SUPPORT.md (hunk 1 above)
Comment (Tech-TTGames) [reply SUPPORT.md:1]: Eh. Text size is alr. Replaced.

**Pull request merged**